.php-cs-fixer.dist.php

```diff
@@ -29,17 +29,8 @@
         '@Symfony' => true,
         '@Symfony:risky' => true,
         'protected_to_private' => false,
-        'native_constant_invocation' => ['strict' => false],
-        'no_superfluous_phpdoc_tags' => [
-            'remove_inheritdoc' => true,
-            'allow_unused_params' => true, // for future-ready params, to be replaced with https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7377
```

Comment:
https://github.com/symfony/symfony/pull/54073#issuecomment-1966087245

OK OK, there is actually a single change in configuration.
instead of using `'allow_unused_params' => true` , we use `'allow_hidden_params' => true` added in linked PR ;)

Reply:
Can you share a link to the PR you're mentioning?

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7743

(the one that is linked in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7377 )